packages/nimble-components/src/tabs/tests/tabs-matrix.stories.ts

```diff
@@ -47,7 +48,7 @@ const component = (
             Tab Two ${() => disabledName}
         </nimble-tab>
         <nimble-tab hidden>Tab Three</nimble-tab>
-        <nimble-tab-panel>Tab content one</nimble-tab-panel>
+        <nimble-tab-panel style="width: 450px;">${loremIpsum}</nimble-tab-panel>
```

Comment:
I think having a chromatic test is a good idea but to me it's adding a lot of noise that makes it difficult to compare the changes between the different matrix tests (the scroll bars are a clunky visual).

Maybe this can just be a separate standalone story like the hiddenTabs or textCustomized stories below

Reply:
Sounds good to me. Separated it into its own story.